samples/nrf9160/nrf_cloud_rest_device_message/README.rst

```diff
@@ -0,0 +1,92 @@
+.. _nrf_cloud_rest_device_message:
+
+nRF9160: nRF Cloud REST Device Message
+######################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The REST Device Message sample demonstrates how to use the `nRF Cloud REST API`_ to send a `Device Message <nRF Cloud Device Messages_>`_ using the SendDeviceMessage endpoint. The d2c topic is used in non-bulk mode.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf9160dk_nrf9160_ns
+
+The sample requires an `nRF Cloud`_ account.
+
+.. note::
+   This sample requires modem firmware v1.3.x or later.
+
+Overview
+********
+
+Pressing the button sends a button press message that can be viewed in the `nRF Cloud`_ device terminal (check the Device to Cloud channel). The device must already be provisioned. However, you can provision the device over REST by enabling :kconfig:`CONFIG_REST_DEVICE_MESSAGE_DO_JITP` and following the instructions printed to UART upon boot.
+
+Limitations
+***********
+
+This sample requires the network carrier to provide date and time to the modem.
+Without a valid date and time, the modem cannot generate JWTs with an expiration time.
+
+User interface
+**************
+
+Once provisioned and connected, each press of the configured button (configured using the :kconfig:`CONFIG_REST_DEVICE_MESSAGE_BUTTON_EVT_NUM` option) generates a device to cloud button press `Device Message <nRF Cloud Device Messages_>`_ over REST.
+
+If you have the option :kconfig:`CONFIG_REST_DEVICE_MESSAGE_DO_JITP` enabled and you press the button when prompted at startup, it will request just-in-time provisioning (JITP) through REST with nRF Cloud.
+This is useful when initially provisioning and associating a device on nRF Cloud.
+You only need to do this once for each device.
+
+The configured LED (:kconfig:`CONFIG_REST_DEVICE_MESSAGE_LED_NUM`) will light once an LTE connection is established and JWT tokens are ready to be generated. Assuming the device is provisioned, this is analogous to being connected to nRF Cloud, though the REST API does not truly employ connections beyond the transient connections inherent to HTTP requests.
```

Comment:
I was trying to communicate that an LTE connection and valid JWTs are the REST API's closest equivalent to being "connected" to nRF Cloud -- but only if the device has already been provisioned.

Maybe it would be easier to delete the sentence outright. Is that fine by you?

Reply:
Maybe it should be simpler, like "REST connections are transient, unlike MQTT connections which are usually long-lived."

Or alternatively, use one of the other LEDs on the 9160DK to blink on and off when a REST transfer occurs, to make this visible to the user.

Reply:
I think I can reasonably do both of those things. I will actually implement blinking tomorrow, though, since i am occupied testing another sample.